### DISCUSSION TITLE
A few configuration questions.

### DISCUSSION THREAD

**Discussion by paolosezart** [wiresock/proxifyre#30] [Ideas]
1. What values can the "logLevel" parameter take in the app-config.json file?

2. How to write applications to the "appNames" parameter, by the name of the window, by the name of the process, and if by the name of the process, do I need to add the .exe extension?

3. If there is no authorization on the proxy server, what should I do with the parameters "username" and "password", delete or can I leave it, since there is no authorization?

4. Does the developer plan to add the ability to run ProxiFyre from a BAT script with parameters, ignoring the app-config.json file?

Thanks.

**Comment by wiresock** [comment]
Hello,

I've addressed your queries in the updated README.MD. Could you elaborate on your vision for `running ProxiFyre from a BAT script with parameters`?

**Comment by paolosezart** [comment]
> Could you elaborate on your vision for `running ProxiFyre from a BAT script with parameters`?

Thank you very much! I have studied the updated README.md, now everything is much clearer.

Let me explain the fourth point by example, let's say I need to quickly run firefox.exe through a proxy, I enter something like:   

```
start ProxiFyre.exe -app firefox.exe -socks5 100.100.100.100:3128 -tcp -udp 
start firefox.exe

start ProxiFyre.exe -app brave.exe -socks5 101.101.101.101:3129 -tcp 
start brave.exe
```

The essence of such manipulations is to quickly change proxies for different programs (browsers are given as an example), because proxy setting is built-in in them.

Or as a variant start with different names app-config.json, example:

```
start ProxiFyre.exe -с app-config1.json
start firefox.exe
start ProxiFyre.exe -с app-config2.json
start brave.exe
```
The main idea is to be able to change the proxy server for an individual program without affecting the others. The explanation is a bit rambling, but I hope I got my point across.
